src/routes/api/v1/search-index/+server.ts

```diff
@@ -15,36 +17,20 @@ const CONTENTFUL_ACTIONS = {
 };
 
 export const POST = async ({ request, locals: { contentfulClient } }) => {
-  authenticateRequest(request);
+  // authenticateRequest(request);
```

Comment:
We'll want to uncomment this before merge, yeah?

Reply:
we will, I'm guessing this was just due to @larsmaxwell getting around not having this set up for local development